settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'Reminder'
```

Comment:
a newline please. Github expects a newline at the end of each file, therefore it will automatically mark it like this.